content/en/logs/guide/lambda-logs-collection-troubleshooting-guide.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Lambda Function Log Collection Troubleshooting Guide
+kind: documentation
+---
+
+## Logs from lambda functions are not visible in Log Explorer page
+
+See the [Datadog-AWS Log integration][1] to configure your environment. If you still do not see your logs, double-check the following points:
+
+There are some common issues that can come up when configuring the AWS Lambda function to forward logs to Datadog. Below are some troubleshooting steps to try if you are using the Lambda function and your logs aren't reaching Datadog. If you continue to have trouble after following these steps, [contact our support team][2] for further assistance.
+
+## Check if logs are reaching the Live Tail
+
+1. Navigate to the [Log Explorer's Live Tail View][3].
+2. In the Search Bar, use a filter to limit the Live Tail View to just the logs coming from your lambda function. Some common search queries are:
+    * By source: the source is often set to `source:aws` or `source:cloudwatch` but you can find other possible sources in the `parse_event_source` function in the [Lambda function][4]. 
+    * By forwarder name: the Lambda function adds a forwardername tag to all the logs it forwards. You can filter on this tag by searching for `forwardername:*` or `forwardername:<function_name>`.
+3. If you see your logs in the Live Tail View but they are not reaching the Log Explorer, this could be due to one of the following reasons:
+    * Logs are too old: we drop logs with a timestamp older than 6 hours.
+    * If your index has any [exclusion filters][5] set up, they may be filtering out your logs.
+
+## Check the Lambda function monitoring tab
+
+From the Lambda Function: 
+
+1. Click the Monitoring tab.
+
+{{< img src="logs/guide/lambda-monitoring-tab.png" alt="Monitoring tab"  style="width:80%;" >}}
+
+2. The monitoring tab displays a series of graphs indicating the following information about your Lambda function: 
+    * invocations
+    * errors
+    * logs
+
+3. If you don't see any data points on the "Invocations" graph, there may be a problem with the triggers you set for your function. See [Manage Your Function Triggers](#manage-your-function-triggers). To get insight into your lambda invocations without using the monitoring tab, see [Viewing Lambda metrics in Datadog](#viewing-lambda-metrics-in-datadog).
+4. If you see data points on the "Error count and success rate" graph, [Check the Lambda function logs](#check-the-lambda-function-logs) to see what error messages are being reported.
+
+## Viewing Lambda metrics in Datadog
```

Comment:
Should this be more of a note ? or a ### rather than a title as the same level as the others?